test/instruments_test.go

```diff
@@ -14,120 +14,201 @@ import (
 )
 
 func TestCreateAndGetInstrument(t *testing.T) {
```

Comment:
Basically the same comments from test/instruments_previous_test apply here.

Reply:
👍